packages/config/config/devices/0x0090/893.json

```
@@ -0,0 +1,175 @@
+					"value": 0
+				},
+				{
+					"label": "Enables the program button",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Secure Screen Control",
+			"description": "The configuration parameter 4 is a read/write parameter with a one-byte field, used to control the secure screen functionality on touch locks only. this parameter is not used on non-touch locks and will be ignored. if set to enabled, the touch screen will display the secure screen functionality when woken. if set to disabled, the touch screen will not display the secure screen functionality when woken",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disables secure screen (touch products only)",
+					"value": 0
+				},
+				{
+					"label": "Enables secure screen (touch products only)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Handing Control",
+			"description": "The configuration parameter 10 is a read/write parameter with a one-byte field. when this parameter is read, it will return the handing of the lock. when this parameter is written, it will run a handing operation. when written with a value of 0x01, it will cause the lock to cycle through locking and unlocking until it determines the correct handing. it is suggested that this operation occur while the door is opened so that no obstructions will be in the way for the operation. after the completion of the handing process, a configuration report message will be sent back to the requestor. it will contain a non-zero value for success or a zero value for failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unknown latch position (read only)",
+					"value": 0
+				},
+				{
+					"label": "Right handed lock (read); Initiate handing process (write)",
+					"value": 1
+				},
+				{
+					"label": "Left handed lock (read only)",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "40",
+			"label": "Factory Default Control",
+			"description": "The configuration parameter 40 is a write only parameter with a one-byte field, used to request the lock to perform a factory reset back to its default state. this control will cause the lock to be removed from the network, placing the lock back to its default state. when a read is issued for this parameter, a 0 will be returned",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No action (get)",
+					"value": 0
+				},
+				{
+					"label": "Perform a full factory reset (set)",
+					"value": 1
+				},
+				{
+					"label": "Perform a modified factory reset (set) - for production only",
+					"value": 2
+				}
```

```suggestion
				{
					"label": "Normal Operation",
					"value": 0
				},
				{
					"label": "Reset to Factory Default",
					"value": 1
				},
				{
					"label": "Perform a modified Factory Reset",
					"value": 2
				}
```
(whatever the modified factory reset is)